coding_introduction/CodingIntroduction.md

```diff
@@ -410,6 +411,17 @@ Answers almost all questions related to GitHub.
 - There is also a freely available [Book on Git](https://git-scm.com/book/en/v2).
 
 
+### Upload SSH Key to `GitHub` 
+Create a new ssh key by calling 
+```bash
+ssh-keygen
+```
+
+Display the public key
+```bash
+cat .ssh/id_rsa.pub
+```
+and copy it. Open the Settings in GitHub and go to the tag SSH and GPG keys under Personal settings. Click on `New SSH key` and paste the key into the respective field. Finally, specify a title and press `Add SSH key` to commit the changes.
```

Comment:
Please add some linebreaks (e.g. each sentence on seperate line, or after 120 chars. Also, put `SSH and GPG keys` in backticks (`) as well. Thanks